frontend/src/metabase/search/types.ts

```diff
@@ -16,9 +16,11 @@ export interface WrappedResult extends SearchResult {
 }
 
 export type TypeFilterProps = SearchModelType[];
+export type VerifiedFilterProps = boolean | undefined;
```

Comment:
Can we get away with not using `undefined` for booleans? Using `undefined` suggests that we treat `undefined` differently from `true` and `false` and that could complicate a really simple boolean checks in many cases.

Reply:
This is one that I'm grappling with - I didn't want to put `false` for two reasons:

* The backend expects `verified` to either be `true` or just not be there at all, and my thinking was that the frontend code should try and reflect the expected backend payload. 
* I think putting `false` here (and in the other comment with `onChange(undefined)`) is confusing. If we were to say `verified=false`, I think someone would come in and say "we're trying to search for unverified items" while the true behavior is that we're trying to get _all_ items.

What do you think? I personally think it's more confusing to put `false` instead of `undefined` because of the API, but I'd appreciate your perspective

Reply:
sound reasonable to keep that in sync with BE. In that case, the type should be `true | undefined` instead. Since `boolean | undefined` implies 3 states are valid
1. `true`
2. `false`
3. `undefined`

Reply:
Ah this is so much better. I'll fix that now.